common/control_files/if-vhost0-dpdk

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+source /opt/contrail/bin/vrouter-functions.sh
+
+if [ ! -L /sys/class/net/vhost0 ]; then
+    vrouter_dpdk_start &>> $LOG
```

Comment:
This file differs only by name of called function (vrouter_dpdk_start instead of insert_vrouter) from the if-vhost0. I think we should minimize copies of the same code as it increases maintenance costs and is more error prone. We should have common vrouter_start/insert function that acts differently on hosts with and without the DPDK.